example/kernelSpecialization/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
Hmm. I copied this from another example, and it seems everywhere in cmakelists of examples there is this emptyish starting line. Is it a typo that got copied around? @psychocoderHPC

Reply:
I do not know why it is there but if it is in all examples I am fine to keep it.
If we like to fix it I would suggest a separate PR to fix it in all examples/tests.

Reply:
Sure, if we fix it should be fixed everywhere. It's actually part of all alpaka cmake files, not just for examples. So I guess it's a formatting style then